frameworks/hdfs/universe/config.json

```diff
@@ -47,6 +47,17 @@
           "default": "serial",
           "enum" : ["parallel", "serial"]
         },
+        "journal_readiness_check": {
+          "type": "object",
+          "description": "Readiness check configuration properties for Journal Nodes.",
```

Comment:
This description is strange.  Maybe, "Determines whether to enable a readiness check for Journal nodes."